packages/admin-app/src/layouts/UnauthenticatedLayout.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, makeStyles } from '@material-ui/core';
+import React from 'react';
+
+import AppBar from '../components/AppBar';
+import Footer from '../components/Footer';
+
+const useStyles = makeStyles(() => ({
+    root: {
+        minHeight: '100vh',
+    },
+}));
+
+export interface DefaultLayoutProps {
+    isLoading?: boolean;
+}
+
+export const AuthenticatedLayout: React.FC<DefaultLayoutProps> = ({ children }) => {
```

Comment:
This class should prolly be UnauthenticatedLayout

Reply:
Great catch, thanks!